### DISCUSSION TITLE
Not sure if its possible, but modifying the way file can be downloaded

### DISCUSSION THREAD

**Discussion by Cisconate** [mozilla/policy-templates#1041] [Q&A]
So I found a thread from a long time ago about ADMX files throwing an error on line 7 column 9.  I opened the new files, along with the ones that were verified working by others. and discovered, if you right click the file, and then select save as, and save as admx file, it LOOKS like it works, however it really downloads a gibberish HTML page.  

Pictures below.  Not sure if this is possible or in your control.  seems odd that you cant download a file this way on github but works elsewhere....

![ADMXSNIP1](https://github.com/mozilla/policy-templates/assets/10426659/ff81be0c-cb5a-40ff-aa50-3b7399ac8d83)
![ADMXSNIP2](https://github.com/mozilla/policy-templates/assets/10426659/8b452484-58d1-43c3-a23d-7da11d30d8e0)

ADMX templates dont support HTML, and they dont support gibberish lol

Thanks!

**Comment by mkaply** [comment]
So the problem is you are download the HTML version of the file from Github.

If you download the official versions of the file from the releases page:

https://github.com/mozilla/policy-templates/releases

You won't have this issue.

Alternatively, click the Raw button in Github - 
![image](https://github.com/mozilla/policy-templates/assets/345868/be1fb7db-dee6-4aaf-aa38-748b4a632bf2)
And the version that is displayed can then be saved.
